lms/djangoapps/sneakpeek_deeplink/tests.py

```diff
@@ -143,3 +155,17 @@ def test_sneakpeek_course_enrollment_closed(self):
         req.path = '/courses/closed/course/run/info'
         self.assertIsNone(self.middleware.process_request(req))
         self.assertNoSneakPeek(req, self.closed_course)
+
+    def test_deleted_course_with_preferences(self):
+        req = self.make_successful_sneakpeek_login_request()
+        req.path = '/courses/deleted/course/run/courseware'
+        self.assertIsNone(self.middleware.process_request(req))
+        self.assertSuccessfulSneakPeek(req, self.course_to_delete)
+
```

Comment:
An extra spaces here makes this block of code look like a new function.  I'm not a stickler on this "rule" but it does make it harder to read in this case.  Adding a comment may be most appropriate since there are 3 sets of tests in this method.  I assume they are related so a one line comment for each block explaining the flow would be helpful.  If they are not related then they should be in different test functions.

I'm not a fan of using the same variable name, req, for the two test blocks in the same method.   Adding a suffix to each one would make this easier to read and help explain what's going on.